UI/src/components/templates/capone.html

```diff
@@ -9,7 +9,7 @@
 
                 <div class="pipeline-toggle btn-group"
 
-                     ng-show="template.hasComponents(ctrl.dashboard, ['Deployment'])">
+                     ng-show="template.hasComponents(ctrl.dashboard, ['SCM'])">
```

Comment:
currently the pipeline and cloud widget are only shown when somebody configures the Deployment widget, this will allow the tabs to be shown on the upper right if SCM widget is configured. I don't think this is needed any more as i created a separate cloud widget